src/lib/util/view.h

```diff
@@ -0,0 +1,154 @@
+// license:BSD-3-Clause
+// copyright-holders:Nathan Woods
+/******************************************************************************
+
+	view.h
+
+	STL container for a view
+
+***************************************************************************/
+
+#pragma once
+
+#ifndef __VIEW_H__
```

Comment:
I'm sick of saying it: this is unacceptable.  **ALL NAMES BEGINNING WITH AN UNDERSCORE OR CONTAINING TWO CONSECUTIVE UNDERSCORES ARE RESERVED IN C++.**
Use something like `MAME_LIB_UTIL_VIEW_H`.